### PR TITLE
save model as .tf & labels/features astype float32

### PR DESCRIPTION
**Pull request by mahsanghani**
save model as .tf for h5 bug in saving keras models

linear model train labels and features astype float32

### CODE REVIEW

**Comment by MarkDaoust** [comment]
I think I'm merging: https://github.com/tensorflow/docs/pull/2295

**Comment by mahsanghani** [comment]
hi can we merge this so the notebook is not failing, thanks! @MarkDaoust
